tests/test_scope.py

```diff
@@ -811,6 +811,31 @@ def test_should_send_default_pii_false(sentry_init):
     assert should_send_default_pii() is False
 
 
+def test_should_send_default_pii_default_false(sentry_init):
+    sentry_init()
+
+    assert should_send_default_pii() is False
+
+
+def test_should_send_default_pii_false_with_dsn_and_spotlight(sentry_init):
+    sentry_init(dsn="http://key@localhost/1", spotlight=True)
+
+    assert should_send_default_pii() is False
+
+
+def test_should_send_default_pii_true_without_dsn_and_spotlight(sentry_init):
+    sentry_init(spotlight=True)
+
+    assert should_send_default_pii() is True
+
+
+def test_should_send_default_pii_false_without_dsn_and_spotlight(sentry_init):
+    sentry_init(spotlight=True, send_default_pii=False)
+
+    # There is now now way to opt out of sending PII to spotlight
+    assert should_send_default_pii() is True
+
```

Comment:
Tehre is now now way to opt out of sending PII to spotlight. If this is just a temporary fix, I think this is OK, but your call @BYK